Lib9c/Action/GoldDistribution.cs

```diff
@@ -37,12 +37,22 @@ public string AddressString
         public static GoldDistribution[] LoadInDescendingEndBlockOrder(string csvPath)
         {
             GoldDistribution[] records;
+#if UNITY_ANDROID
+            UnityEngine.WWW www = new UnityEngine.WWW(csvPath);
+            while (!www.isDone)
+            {
+                // wait for data load
+            }
+            StreamReader streamReader = new StreamReader(new MemoryStream(www.bytes), System.Text.Encoding.Default);
+            CsvReader csvReader = new CsvReader((TextReader)streamReader, CultureInfo.InvariantCulture);
```

Comment:
Can it dispose `streamReader` and `csvReader`?

Reply:
Since this code had been cherry-picked from https://github.com/virtuosgames/lib9c/commit/0917aab714be1cb75d119a8dd8e79876267841c0 (by hand 😅), I can't make sure disposing of is okay on android. cc @yezihong

Reply:
I still can't confirm in strict way. but at least on my device, https://github.com/planetarium/lib9c/pull/1714/commits/81533dbe8f2bf0b6d695cacd61ef43445d7ee7f7 doesn't seem to cause problem on build. 

PTAL @boscohyun

Reply:
Thanks for checking!